httpcore5/src/test/java/org/apache/hc/core5/net/TestURIBuilder.java

```diff
@@ -617,7 +618,7 @@ void testHttpHost() throws Exception {
 
     @Test
     void testSetHostWithReservedChars() throws Exception {
-        final URIBuilder uribuilder = new URIBuilder();
+        final URIBuilder uribuilder = new URIBuilder().setEncodingPolicy(URIBuilder.EncodingPolicy.ALL_RESERVED);
```

Comment:
@arturobernalg I think this should no longer be necessary. `ALL_RESERVED` is default now.

Reply:
done